bitnami/spark/templates/statefulset-master.yaml

```diff
@@ -142,13 +142,15 @@ spec:
           args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
           {{- end }}
           ports:
-            {{- if .Values.security.ssl.enabled }}
-            - name: https
-            {{- else }}
+            {{- if not .Values.security.ssl.enabled }}
```

Comment:
I now that is the same at the end, but would you mind changing the order to avoid checking in negative ?.
```
          ports:
            {{- if .Values.security.ssl.enabled }}
            - name: https
              containerPort: {{ .Values.master.webPortHttps }}
              protocol: TCP
            {{- else }}
            - name: http
              containerPort: {{ .Values.master.webPort }}
              protocol: TCP
            {{- end }}
```

Reply:
Changed in 059e4eb11